src/ontoform/transformers/gnomad.py

```diff
@@ -0,0 +1,16 @@
+import gzip
+from typing import BinaryIO
+
+from loguru import logger
+
+from ontoform.util import SupportedFormats
+
+
+def transform(src: BinaryIO, dst: BinaryIO, format: SupportedFormats) -> None:
+    """Take for e.g. a bgzipped file and write it to a gzip file."""
+    logger.info(
+        f"transforming {src.name} to {dst.name}, ignoring format: {format.name}"
+    )
+    with gzip.open(src, "rb") as file:
+        with gzip.open(dst, "wb") as gzip_file:
```

Comment:
```suggestion
    with gzip.open(src) as file:
        with gzip.open(dst, 'wb') as gzip_file:
```

`rb` is alreay the default for [`gzip.open()`](https://docs.python.org/3/library/gzip.html#gzip.open), also, we use single quotes in strings. I need to write a vscode config that enables ruff by default for the project so we get autoformat on save.